tests/run-tests-compose.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+source .env
+
+export PODMAN_IGNORE_CGROUPSV1_WARNING=1
+
+cleanup () {
+    echo "cleaning"
+    podman-compose -f tests/compose.yml down --volumes --remove-orphans --timeout 0
+    echo "podman environment cleaned"
+}
+
+# cleanup
```

Comment:
what's this comment for?

Reply:
Removed.  The function `cleanup` was being tested.